packages/interfaces/src/kobo/IncidentDto.ts

```diff
@@ -86,6 +86,8 @@ export class IncidentDto {
   @IsOptional() @IsNumberString() readonly 'group_pk03i04/group_ub2hp14/CraftDam'?: string;
   @IsOptional() @IsNumberString() readonly 'group_gh6ag70/group_na4yi04/CropAff'?: string;
   @IsOptional() @IsNumberString() readonly 'group_gh6ag70/group_na4yi04/CropDam'?: string;
+  @IsOptional() @IsNumberString() readonly 'group_gh6ag70/group_az7bv13/integer_ws0la67'?: string;
```

Comment:
@wadhwamatic I think they should update these names before they get integrated in other apps to stay consistent with the rest

Reply:
agreed. I'll have them update it in kobo